mergin/client.py

```diff
@@ -205,19 +205,26 @@ def _do_request(self, request):
         try:
             return self.opener.open(request)
         except urllib.error.HTTPError as e:
-            if e.headers.get("Content-Type", "") == "application/problem+json":
-                info = json.load(e)
-                err_detail = info.get("detail")
+            server_code = None
+            server_response = None
+            if e.headers.get("Content-Type", "") == "application/json":
+                server_response = json.load(e)
+                err_detail = server_response.get("detail")
+                server_code = server_response.get("code")
```

Comment:
How does it work - the `code` is only transferred in this case, but not in `application/problem+json` case?

(probably I am just not familiar enough with various server error responses :slightly_smiling_face: )

Reply:
We checked the server side and it seems we are returning simple `application/json` and not the one with `problem+json`. Here is a ticket https://github.com/MerginMaps/server-private/issues/2459, I'd prefer if we did not check the `Content-Type` for now. The mobile app does not do it either.